### DISCUSSION TITLE
请教一下openwrt部署了Alist，定时重启之后会有错误提示，不知道哪里的问题

### DISCUSSION THREAD

**Discussion by xinxinenjoy** [AlistGo/alist#2835] [Q&A]
阿里云
Post"https://auth.aliyundrive.com/v2/account/token":x50状9: certificate has expired or is not yet valid: current time 2感000-01-01T00:00:23Zis before 2022-04-27T04:02:02Z
115
Get"https://passportapi.115.com/app/1.0/web/1.0/check/s状 so?=946684824330": x509: certificate has expired or is态:not vet valid: current time 2000-01-01T00:00:24Zis befole2022-11-29T00:00:00Z
像是提示的什么时间错误？
另外打开WEB界面里的ALIST日志之后，发现读取的时间跟现实时间是不符的，但是路由器的系统时间并无问题
出现这个提示之后如果挂载的有“阿里云分享”，那么云分享的端口不会失效，我自己的盘会打不开，提示“failed get objs: failed to list objs: The input parametershare id, drive id is not valid. get file request drive idand share id are mutually exclusive. and must be passed only one”，ID跟分享ID互斥

**Comment by github-lk1** [comment]
想问一下在软路由上部署，是可以在任意地方都能访问到吗

**Reply by cheneylaw** [reply]
那你的软路由得能让外部访问得到才行，例如先部署VPN或者Zerotier就可以

**Comment by xinxinenjoy** [comment]
回复一下，之前这个问题是openwrt插件的毛病，现在我的方案是在爱快软路由里，用docker部署，方便很多
关于上边朋友问的外网访问：1.首先你要有公共网，直接IP访问或者对应自己的域名。2.在路由器里把Alist对应的IP地址的端口映射打开。这样就可以在外网随时访问了
然后牵扯到一个进阶的问题：挂载的网盘如果支持302定向，那么你在外网访问的时候就不用经过家里中转数据。如果选择的是本地缓存，那就需要家里的路由器先下载，再上传给你外网访问的设备（也就是外边访问的速度跟你家里的上行带宽挂钩）。
